sci-libs/fftw/fftw-3.3.8.recipe

```diff
@@ -15,42 +15,74 @@ HOMEPAGE="http://www.fftw.org/"
 COPYRIGHT="2003, 2007-2011 Matteo Frigo
 	2003, 2007-2011 Massachusetts Institute of Technology"
 LICENSE="GNU GPL v2"
-REVISION="1"
+REVISION="2"
 SOURCE_URI="http://www.fftw.org/fftw-$portVersion.tar.gz"
 CHECKSUM_SHA256="6113262f6e92c5bd474f2875fa1b01054c4ad5040f6b0da7c03c98821d9ae303"
 SOURCE_DIR="fftw-$portVersion"
 
 ARCHITECTURES="x86_gcc2 x86 x86_64"
-SECONDARY_ARCHITECTURES="x86_gcc2 x86"
+SECONDARY_ARCHITECTURES="?x86_gcc2 x86"
 
 libVersion="3.5.8"
 libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
 
 PROVIDES="
 	fftw$secondaryArchSuffix = $portVersion compat >= 3
 	cmd:fftw_wisdom$secondaryArchSuffix
-	cmd:fftw_wisdom_to_conf$secondaryArchSuffix
 	cmd:fftwf_wisdom$secondaryArchSuffix
+	cmd:fftwl_wisdom$secondaryArchSuffix
+	cmd:fftwq_wisdom$secondaryArchSuffix
+	cmd:fftw_wisdom_to_conf$secondaryArchSuffix
```

Comment:
I think we should drop the suffix for x86_gcc2.

Reply:
Do you mean: "drop `$secondaryArchSuffix` for the `cmd:` runtimes" ?

Since the recipe builds on all primary arches, what we _could_ do, instead, is drop some of the `cmd:` runtimes.
But I think the PR is better as it is right now.

Reply:
Oh, my bad, I didn't notice. I think we should drop those `cmd:` for secondary architectures, their functions doesn't appear to deviate between architecture to me.

Reply:
I don't have a strong opinion on whether or not we should drop the `cmd:` runtimes for secindary arches.
I'd say that keeping them **_might_** help x86_gcc2h users who could choose to call the `-x86` versions, as these have access to the quad libs, `libfftw3q{,_threads}.so`, which are not available on x86_gcc2 primary arch.

So, we could choose to drop the `cmd:` runtimes only for all secondary arch builds on platforms x86_64 (and x86), and keep them for x86 secondary arch of x86_gcc2h.

So, to summarize, there are 3 scenari:
1) Keep all `cmd:` in all cases. (That's what we have right now in the PR.)
2) Keep the `cmd:` on secondary arch if `$targetArchitecture" = x86_gcc2.
3) Drop all `cmd:` on all secondary architectures.

I don't know which is best. All I know is that solution 1 covers all possible needs.